import-wizard - new UI

chrtannus opened the pull request:
**General information**

Associated issues: #78

**Checklist**

Author:

- [X] One or more reviewers have been assigned.
- [x] Automated tests have been included in this pull request, if possible, for the new feature(s) or bug fix.
- [X] The associated GitHub issues are included (above).
- [X] Notes have been included (below).

Reviewers:

- [x] All automated checks are passing (green check next to latest commit).
- [x] At least one reviewer has signed off on the pull request.  Reviewers have two business days to review the pull request, after which the author may merge in the pull request unilaterally.


**Notes**

- Changes the first wizard page (the list of wizards are now coloured buttons with SVG logos).
- The **Cy3 Import** wizard now has only one step.
- Adds the **Excel Import** wizard.
- It's already using the new UI (dark theme), but there is one issue:
  - In **Excel Import**, after you drop an "edge table" file, the dialog colours revert to the standard 'white' theme. It seems that this is caused when a _cy.js_ **Popper** (edge or node data table) is created by doing this (see `excel-import-wizard.js`):
   
      `div.innerHTML = ReactDOMServer.renderToStaticMarkup(/*...*/)`
    
      See image below:
  
      <img width="612" alt="import_dialog-wrong_colors" src="https://user-images.githubusercontent.com/989686/138958857-e58c8488-aeb9-44f1-bc2e-149037d8d81f.png">

maxkfranz commented:
@chrtannus, do you want to merge in your PR before #97?  If there are any merge conflicts between the PRs, I can sort them out.

The package lock file conflict can be resolved by deleting the file and then running `npm i`.  That may result in slightly newer patch versions on some dependencies, but that's generally fine.

chrtannus commented:
@maxkfranz I merged develop into my branch first and sorted out the conflicts. Also fixed a couple of other issues and refactored it a little bit. I think it's good to merge now.

maxkfranz commented:
Great!  Let's merge it in
